src/core.h

```diff
@@ -361,6 +364,7 @@ void stream_enable_rtp_timeout(struct stream *strm, uint32_t timeout_ms);
 int  stream_jbuf_reset(struct stream *strm,
 		       uint32_t frames_min, uint32_t frames_max);
 bool stream_is_ready(const struct stream *strm);
+int stream_decode(struct stream *strm);
```

Comment:
stream_decode looks like a leftover from the jbuf pull request

Reply:
Oops. You're right!